nginx.conf.tmpl

```diff
@@ -0,0 +1,32 @@
+server {
+  listen ${HTTP_PORT};
+  server_name ${HOSTNAME};
+  access_log off;
+
+  location / {
+    proxy_pass ${YOTTER_ADDRESS};
+    proxy_set_header Host $host;
+    proxy_set_header X-Real-IP $remote_addr;
+    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
```

Comment:
Does Yotter really need the remote IP? On my instance, it works perfectly fine without it.

Reply:
You should remove it :)
I'm afraid that it could potentially leak the IP of a user in a misconfigured instance.

Reply:
That's a good point! Updated